csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -216,37 +398,37 @@ internal static class NativeMethods
         #endregion
 
         #region RunOptions API
-        [DllImport(nativeLib, CharSet = charSet)]
-        public static extern IntPtr /*(OrtStatus*)*/ OrtCreateRunOptions( out IntPtr /* OrtRunOptions** */ runOptions);
```

Comment:
No white-space between "(" and "out".  CTRL+k+d should fix minor formatting issues.

Reply:
Not sure how that happened, it was formatting it for everything else. Fixed